.github/ISSUE_TEMPLATE/new_language_or_locale.md

```diff
@@ -0,0 +1,20 @@
+---
+name: "New language or locale"
+about: "Please create voice packs for my language or locale!"
+title: "Add language/locale: [YOUR LANGUAGE OR LOCALE]"
+---
+
+## Description
+
+I'd like to be able to download voice packs in the following language(s) and locale(s):
+
+<!-- Please list the languages or locales that you'd like to use. Example: `fr_BE`, Japanese, Spanish (Colombia) and add them to the issue title as well. -->
```

Comment:
Maybe a link to a list of lang_locales if you know of one offhand? It's not critical, but it does saves a step if we know that right from the start.

Reply:
I can't find any obviously good reference, but I think that linking to the locale codes that Azure Speech Studio supports is not a bad option since it's likely going to be the TTS in charge. I'll find an adequate link.

Reply:
This [link](https://learn.microsoft.com/en-us/azure/ai-services/speech-service/language-support?tabs=tts#supported-languages) points at the section of tbe Azure docs that:

- is specific to TTS
- lists the supported locales
- shows the BCP-47 codes for them (e.g. `es_AR`)

That will do I think. The standards that BCP-47 refers to don't define the codes themselves, only the rules to create those codes (as standards tend to do).

Reply:
Yup, that will do perfectly. Thanks! :)